docs/ivys.js

```diff
@@ -78,9 +78,9 @@ Product.prototype.removeTag = function () {
 var ProductCollection = function () {
     "use strict";
     var self = this;
-    self.all_products = {};
-    self.favorite_products = {};
-    self.all_tags = {};
-    self.selected_tags = {};
-    self.selected_products = {};
+    self.all_products = {}; // sku number : 'product name' ?
+    self.favorite_products = {}; // sku number : 'product name' ?
+    self.all_tags = {}; // 'tag-name' : 'tag-name' ?
```

Comment:
To know what the keys and values should be we need to figure out how and why these attributes will be used at all. Where is `all_tags` going to be used? What do you need to do with it? Same for the others.

In cases where all we want to do is check whether some tag is part of the set, it doesn't really matter much what the value is.

If we're trying to look up a value using some key then that answers the question for us.

Probably need to decide what the canonical piece of information that identifies a product will be. SKU I assume? If so then use that everywhere.